tests/qos/test_qos_sai.py

```diff
@@ -540,6 +540,9 @@ def testQosSaiLosslessVoq(
             Raises:
                 RunAnsibleModuleFail if ptf test fails
         """
+        if not get_src_dst_asic_and_duts['single_asic_test']:
+             pytest.skip("This test is not applicable when "
+                 "src and dst ASICs are different.")
```

Comment:
update the comment here.

Reply:
Changed it to: "This test needs to be revisited later, for the case where src and dst ASICs are different."